### PR TITLE
Add use sdk blendshape mapping functionality

### PR DESCRIPTION
**Pull request by AndreiEvstratenko**
# Pull Request

## Description

Use sdk mapping functionality. Update imports with the new version of sdk

### CODE REVIEW

File: examples/ARKitFaceFilterDemo/ARKitFaceFilterDemo/VirtualFaceContent/RemoteFaceFilter.swift [hunk 1]
```diff
@@ -9,7 +9,8 @@
 import Foundation
 import SceneKit
 import ARKit
-import GLTFSceneKit
+import SvrfGLTFSceneKit
```
Comment (brentchow) [inline examples/ARKitFaceFilterDemo/ARKitFaceFilterDemo/VirtualFaceContent/RemoteFaceFilter.swift]: Why are we importing this? The SDK should handle glTF to SceneKit conversion

File: examples/ARKitFaceFilterDemo/ARKitFaceFilterDemo/VirtualFaceContent/RemoteFaceFilter.swift [hunk 2]
```diff
@@ -37,28 +38,13 @@ class RemoteFaceFilter: SCNNode, VirtualFaceContent {
                 let modelSource = GLTFSceneSource(url: glbModelUrl)
```
Comment (brentchow) [inline examples/ARKitFaceFilterDemo/ARKitFaceFilterDemo/VirtualFaceContent/RemoteFaceFilter.swift]: This should be in `SvrfSDK.getHead()`

File: examples/ARKitFaceFilterDemo/ARKitFaceFilterDemo/ViewController.swift [hunk 3]
```diff
@@ -63,11 +63,19 @@ class ViewController: UIViewController, ARSCNViewDelegate, ARSessionDelegate, UI
...
+        // Init RemoteFaceFilter
+        let remoteFaceFilter = RemoteFaceFilter()
+        self.contentUpdater.virtualFaceNode = remoteFaceFilter
+        
         // Set hook to create a new RemoteFaceFilter when a face filter is selected from the results
         searchView.selectedItemDidChange = { [unowned self] (_ media: Media) -> Void in
             if let glbUrl = media.files?.glb {
                 // Set the current filter to be the selected Face Filter
-                self.contentUpdater.virtualFaceNode = RemoteFaceFilter(fromUrl: glbUrl)
+//                self.contentUpdater.virtualFaceNode = RemoteFaceFilter(fromUrl: glbUrl)
```
Comment (brentchow) [inline examples/ARKitFaceFilterDemo/ARKitFaceFilterDemo/ViewController.swift]: remove commented out code

Comment (AndreiEvstratenko) [reply examples/ARKitFaceFilterDemo/ARKitFaceFilterDemo/ViewController.swift]: done

**Pull request merged**